src/components/Admin/UserPage.tsx

```diff
@@ -99,7 +99,7 @@ export const UserPage: FC<UserPageProps> = ({ mode = EditMode.readOnly, user, ti
 
   const handleSave = (user: UserModel) => {
     // Convert UserModel to UserInput
-    const { id: userID, profile, memberof, ...rest } = user;
+    const { id: userID, profile, ...rest } = user;
```

Comment:
`memberof` should not be present in the user input, will fail the GraphQL validation.
Saving user will throw an error:
`Field "memberof" is not defined by type "UserInput".`

Reply:
done